api/src/test/scala/hmda/api/model/ModelGenerators.scala

```diff
@@ -70,4 +78,41 @@ trait ModelGenerators {
     } yield FilingDetail(filing, submissions)
   }
 
+  implicit def agencyGen: Gen[Agency] = {
+    Gen.oneOf(
+      CFPB,
+      FDIC,
+      FRS,
+      HUD,
+      NCUA,
+      OCC
+    )
+  }
+
+  implicit def institutionTypeGen: Gen[InstitutionType] = {
+    Gen.oneOf(
+      Bank,
+      CreditUnion,
+      SavingsAndLoan,
+      NonDepositInstType,
+      NoDepositTypeInstType
+    )
```

Comment:
I prefer `Gen.oneOf(InstitutionType.values)` (assuming it works) so I won't have to remember to modify it in #438.
Same deal for `Agency`, `ExternalIdType`, and any others I missed.